test/test_ext.jl

```diff
@@ -0,0 +1,25 @@
+using DashBase
+using Test
+using Plots
+plotlyjs()
+
+@testset "PlotlyJS" begin
+    pl = @test_nowarn DashBase.to_dash(plot(1:5))
+    @test pl isa PlotlyJS.SyncPlot
```

Comment:
@etpinard do these need to return something readable by JSON3 or is it enough, that they return something that has a `to_dash` method?